.github/workflows/dockers_test.yaml

```diff
@@ -24,3 +24,7 @@ jobs:
       - name: Run unit tests for actions-plan-preview
         working-directory: dockers/actions-plan-preview
         run: go test ./...
+
+      - name: Run unit tests for actions-gh-release
+        working-directory: dockers/actions-gh-release
+        run: go test ./...
```

Comment:
Could you fix the rename of this action from `name: Dockers Test` to `name: Test`

Reply:
That make sense!
and what do you think about also renaming of that job name from `Unit Tests` to `Docker Unit Tests`

Reply:
@knanao You are right! Let's do it.